zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -725,6 +725,18 @@ void permissionError(NotebookSocket conn, String op, String userName, Set<String
             .toString())));
   }
 
+  private void cronExecutingUserUpdatePermissionError(NotebookSocket conn, String op, String user,
+                                                      String cronExecutingUser)
+      throws IOException {
```

Comment:
this is a bit hard to read - could it fit after the last line?

Reply:
@felixcheung This was fixed at https://github.com/apache/zeppelin/pull/2671/commits/f4f53c5e9d65a31d57cadc86820a96f385cedee7. Thanks.